src/karapace/karapace_all.py

```diff
@@ -21,6 +23,37 @@ class KarapaceAll(KafkaRest, KarapaceSchemaRegistryController):
     pass
 
 
+def _configure_logging(*, config: Config) -> None:
+    log_level = config.get("log_level", "DEBUG")
+    log_format = config.get("log_format", "%(name)-20s\t%(threadName)s\t%(levelname)-8s\t%(message)s")
+
+    root_handler: logging.Handler | None = None
+    log_handler = config.get("log_handler", None)
+    if "systemd" == log_handler:
+        from systemd import journal
+
+        root_handler = journal.JournalHandler(SYSLOG_IDENTIFIER="karapace")
+    if "stdout" == log_handler or log_handler is None:
```

Comment:
nit: `elif` to highlight that this is an alternative option other than the first entry?

Reply:
Done.